content/en/logs/guide/correlate-your-full-stack-data.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Correlate your full stack logs, traces and views
+kind: guide
+aliases:
+  - /logs/guide/xxx
+further_reading:
+  - link: /logs/guide/logs-rbac/
+    tag: Documentation
+    text: "Set up Roles Based Access Controls (RBAC) for Log Management"
+  - link: /agent/logs/advanced_log_collection
+    tag: Documentation
+    text: "Filter and Redact logs with Advanced Log Collection"
+  - link: /agent/guide/autodiscovery-management/
+    tag: Documentation
+    text: "Exclude containers from Log Collection with Autodiscovery"
+
+---
+
+## Overview
+
+With [unified service tagging][1] you already have high level correlation capabilities. But sometimes the starting point of your investigation is a single log or a single trace. Correlating them with other data gives context to estimate business impact and find root causes in a few clicks.
+
+{{< img src="logs/guide/correlate-your-full-stack-data/full-stack-cover.png" alt="Full stack correlation" style="width:80%;" >}}
+
+Correlating your logs also eases [aggressive sampling strategy based on Trace ID][2] without losing entity-level consistency.
+
+This guide walks you through the steps you should take to correlate your full stack logs, traces and view:
+
+1. [Correlate application logs](#correlate-application-logs)
+2. [Correlate proxy logs](#correlate-proxy-logs)
+3. [Correlate database logs](#correlate-database-logs)
+4. [Correlate queuing logs](#correlate-queuing-logs)
+5. [Test the whole correlation from a Synthetic test](#test-the-whole-correlation-from-a-synthetic-test)
+6. [Correlate browser logs](#correlate-browser-logs)
+7. [Correlate RUM views](#correlate-rum-views)
+
+Note: Depending on your use case, you may skip steps. Steps dependant from others are explicit.
```

Comment:
I'd rather phrase it in a more actionnable way, like
"Correlate logs with application (1) offers extensive visibility across your stack, but some very specific use-cases require to prolong the correlation deeper to your stack (2, 3, 4, 6, 7)".

Also, it feels like the synthetic test section is pretty much in the middle of nothing. My 2cts, let's add a whole "Leverage correlation with Synthetics Tests" section after. And mention Browser Tests as well, to highlight RUM part of the story as well.